cli/medperf/entities/report.py

```diff
@@ -52,71 +59,14 @@ def set_results(self, results):
         self.results = results
 
     @classmethod
-    def all(
-        cls, local_only: bool = False, mine_only: bool = False
-    ) -> List["TestReport"]:
-        """Gets and creates instances of test reports.
-        Arguments are only specified for compatibility with
-        `Entity.List` and `Entity.View`, but they don't contribute to
-        the logic.
-
-        Returns:
-            List[TestReport]: List containing all test reports
-        """
-        logging.info("Retrieving all reports")
-        reports = []
-        tests_folder = config.tests_folder
-        try:
-            uids = next(os.walk(tests_folder))[1]
-        except StopIteration:
-            msg = "Couldn't iterate over the tests directory"
-            logging.warning(msg)
-            raise RuntimeError(msg)
-
-        for uid in uids:
-            local_meta = cls.__get_local_dict(uid)
-            report = cls(**local_meta)
-            reports.append(report)
-
-        return reports
-
-    @classmethod
-    def get(cls, report_uid: str) -> "TestReport":
-        """Retrieves and creates a TestReport instance obtained the user's machine
-
-        Args:
-            report_uid (str): UID of the TestReport instance
-
-        Returns:
-            TestReport: Specified TestReport instance
-        """
-        logging.debug(f"Retrieving report {report_uid}")
-        report_dict = cls.__get_local_dict(report_uid)
-        report = cls(**report_dict)
-        report.write()
-        return report
-
-    def todict(self):
-        return self.extended_dict()
-
-    def write(self):
-        report_file = os.path.join(self.path, config.test_report_file)
-        os.makedirs(self.path, exist_ok=True)
-        with open(report_file, "w") as f:
-            yaml.dump(self.todict(), f)
-        return report_file
+    def all(cls, unregistered: bool = False, filters: dict = {}) -> List["Entity"]:
+        assert unregistered, "Reports are only unregistered"
+        assert filters == {}, "Reports cannot be filtered"
+        return super().all(unregistered=True, filters={})
 
     @classmethod
-    def __get_local_dict(cls, local_uid):
-        report_path = os.path.join(config.tests_folder, str(local_uid))
-        report_file = os.path.join(report_path, config.test_report_file)
-        if not os.path.exists(report_file):
-            raise InvalidArgumentError(
-                f"The requested report {local_uid} could not be retrieved"
-            )
-        with open(report_file, "r") as f:
-            report_info = yaml.safe_load(f)
-        return report_info
+    def get(cls, report_uid: str, local_only: bool = False) -> "TestReport":
+        return super().get(report_uid, local_only=True)
```

Comment:
```suggestion
    def get(cls, uid: str, local_only: bool = False) -> "TestReport":
        """Gets an instance of the TestReport. ignores local_only inherited flag as TestReport is always a local entity.
        Args:
            uid (str): Report Unique Identifier
            local_only (bool): ignored. Left for aligning with parent Entity class
        Returns:
            TestReport: Report Instance associated to the UID
        """
        return super().get(uid, local_only=True)
```
(1) to highlight it's not a bug and `local_only` is ignored conciously,
(2) still keep the same function signature (in case if we once want to make `cls.get(uid=entity_uid)` that would look proper from the IDE side but would fail on TestReport entities)